test/user.e2e-spec.ts

```diff
@@ -0,0 +1,40 @@
+import * as request from "supertest";
+import { Test, TestingModule } from "@nestjs/testing";
+import { INestApplication } from "@nestjs/common";
+import { AppModule } from "../src/app.module";
+
+describe("UserController (e2e)", () => {
+	let app: INestApplication;
+
+	beforeEach(async () => {
+		const moduleFixture: TestingModule = await Test.createTestingModule({
+			imports: [AppModule]
+		}).compile();
+
+		app = moduleFixture.createNestApplication();
+		await app.init();
+	});
+
+	it("/user/1 (GET)", () => request(app.getHttpServer())
+		.get("/user/1")
+		.expect(200)
+		.expect(res => {
+			if (res.body.id !== 1) {
+				return false;
+			}
+
+			if (res.body.name !== "testuser") {
+				return false;
+			}
+
+			return true;
```

Comment:
I’m a bit confused what's going on here. Are you just comparing individual properties and if one doesn't match the return fails the test rather than just comparing it to a full response body?